apps/dashboard/src/components/activity/activity-job-item.tsx

```diff
@@ -142,12 +143,22 @@ function getStatusMessage(job: IActivityJob): string | React.ReactNode {
       return '';
     case StepTypeEnum.DIGEST:
       if (job.status === JobStatusEnum.COMPLETED) {
+        if ((job.digest as IDigestTimedMetadata).timed?.untilDate) {
+          return `Digested events until scheduled time${job.scheduleExtensionsCount && job.scheduleExtensionsCount > 0 ? `, extended to subscriber schedule` : ''}`;
+        }
```

Comment:
`untilDate` is calculated during workflow execution for the scheduled digest/delay